### PR TITLE
Handle upload errors

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/9002

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-runtime/src/components/UploadInfo.vue [hunk 1]
```diff
@@ -573,26 +573,35 @@ export default defineComponent({
...
-        }
-        if (error.message.includes('precondition failed:')) {
-          errorMessage = this.$gettext('Parent folder does not exist')
+      if (!error) {
+        return
+      }
+
+      //TODO: Remove extraction code as soon as https://github.com/tus/tus-js-client/issues/448 is solved
+      const formatErrorMessageToObject = (errorMessage) => {
+        let responseCode = errorMessage.match(/response code: (\d+)/)?.[1]
```
Comment (fschade) [inline packages/web-runtime/src/components/UploadInfo.vue:582]: any reason why `let` and not `const`

**Review by fschade — APPROVED** [review]
LGTM 🚀

File: packages/web-runtime/src/components/UploadInfo.vue (hunk 1 above)
Comment (JanAckermann) [reply packages/web-runtime/src/components/UploadInfo.vue:582]: sure, will be parsed to int (some lines under it), and parsing something else like null or undefined will result in NaN

**Review by lookacat — APPROVED** [review]
👍🏼

**Review by kulmann — DISMISSED** [review]
Changelog item please 😁

**Pull request merged**